build.sbt

```diff
@@ -273,6 +273,160 @@ lazy val `example-routeguide-client` = project
+
+lazy val `seed-server-app` = project
+  .in(file("modules/examples/seed/server/modules/app"))
+  .settings(coverageEnabled := false)
+  .settings(noPublishSettings)
+  .dependsOn(server, `seed-server-process`, `seed-config`)
+
+//////////////////////////
+////      Server      ////
+//////////////////////////
+
+lazy val allServerModules: Seq[ProjectReference] = Seq(
+  `seed-server-common`,
+  `seed-server-protocol-avro`,
+  `seed-server-protocol-proto`,
+  `seed-server-process`,
+  `seed-server-app`
+)
+
+lazy val allServerModulesDeps: Seq[ClasspathDependency] =
+  allServerModules.map(ClasspathDependency(_, None))
+
+lazy val `seed-server` = project
+  .in(file("modules/examples/seed/server"))
+  .settings(coverageEnabled := false)
+  .settings(noPublishSettings)
+  .aggregate(allServerModules: _*)
+  .dependsOn(allServerModulesDeps: _*)
+addCommandAlias("runAvroServer", "seed_server/runMain example.seed.server.app.AvroServerApp")
+addCommandAlias("runProtoServer", "seed_server/runMain example.seed.server.app.ProtoServerApp")
+
+//////////////////////////
+////  Client Modules  ////
+//////////////////////////
+
+lazy val `seed-client-common` = project
+  .in(file("modules/examples/seed/client/modules/common"))
+  .settings(coverageEnabled := false)
+  .settings(noPublishSettings)
+
+lazy val `seed-client-process` = project
+  .in(file("modules/examples/seed/client/modules/process"))
+  .settings(coverageEnabled := false)
+  .settings(noPublishSettings)
+  .settings(exampleSeedLogSettings)
+  .dependsOn(netty, fs2, `seed-client-common`, `seed-server-protocol-avro`, `seed-server-protocol-proto`)
+
+lazy val `seed-client-app` = project
+  .in(file("modules/examples/seed/client/modules/app"))
+  .settings(coverageEnabled := false)
+  .settings(noPublishSettings)
+  .settings(exampleSeedClientAppSettings)
+  .dependsOn(`seed-client-process`, `seed-config`)
+
+//////////////////////////
+////      Client      ////
+//////////////////////////
+
+lazy val allClientModules: Seq[ProjectReference] = Seq(
+  `seed-client-common`,
+  `seed-client-process`,
+  `seed-client-app`
+)
+
+lazy val allClientModulesDeps: Seq[ClasspathDependency] =
+  allClientModules.map(ClasspathDependency(_, None))
+
+lazy val `seed-client` = project
+  .in(file("modules/examples/seed/client"))
+  .settings(coverageEnabled := false)
+  .settings(noPublishSettings)
+  .aggregate(allClientModules: _*)
+  .dependsOn(allClientModulesDeps: _*)
+addCommandAlias("runAvroClient", "seed_client/runMain example.seed.client.app.AvroClientApp")
+addCommandAlias("runProtoClient", "seed_client/runMain example.seed.client.app.ProtoClientApp")
+
+/////////////////////////
+////       Root       ////
+/////////////////////////
+
+lazy val allSeedModules: Seq[ProjectReference] = Seq(
+  `seed-shared`,
+  `seed-client`,
+  `seed-server`
+)
+
+lazy val allSeedModulesDeps: Seq[ClasspathDependency] =
+  allSeedModules.map(ClasspathDependency(_, None))
+
+lazy val seed = project
```

Comment:
Any reason for having this module?

Reply:
One module to rule them all

Reply:
![giphy](https://user-images.githubusercontent.com/720923/56226443-81ff2b80-6073-11e9-81d2-49d801e47c0f.gif)